executors/injectexec/injectexec_test.go

```diff
@@ -0,0 +1,100 @@
+//
+// Copyright (c) 2018 The heketi Authors
+//
+// This file is licensed to you under your choice of the GNU Lesser
+// General Public License, version 3 or any later version (LGPLv3 or
+// later), or the GNU General Public License, version 2 (GPLv2), in all
+// cases as published by the Free Software Foundation.
+//
+
+package injectexec
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/heketi/heketi/executors/cmdexec"
+	"github.com/heketi/heketi/executors/mockexec"
+
```

Comment:
can we remove this extra line?

Reply:
I would rather just fix the ordering mistake here.